Makefile

```diff
@@ -1,10 +1,17 @@
 EXTVERSION   = 0.0.1
-REGRESS      = 01_general 02_partitioning
 REGRESS_OPTS = --inputdir=test
 PGFILEDESC   = "pg_anonymize - perform data anonymization transparently on the database"
 
 PG_CONFIG = pg_config
 
+REGRESS      = 01_general \
+               02_partitioning
+ifneq ($(MAJORVERSION), 10)
```

Comment:
unfortunately this cannot work.  The $(MAJORVERSION) is only known after including $(PGXS).  The regression tests didn't spot it as pg10 is now EOL, but I'd rather not drop support for older release unless it makes our life really hard.